### DISCUSSION TITLE
Autistim Hypersensitivity to light - Eye Comfort GitHub Background

### DISCUSSION THREAD

**Discussion by Bless360o** [community/community#56523] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

Haiii gais ❤️ Blessings worldwide 🎐 🗺️
Dark theme helps a lot and less blue light will help a lot more. 
Just that the brightness makes it difficult to read and use GitHub especially at nighttime or in dark conditions
Thank YOU so so much this will really help 😊
Stay BLESSED and Hugs from Costa Rica 🇨🇷🦋

**Comment by Myndex** [comment, marked as answer]
Yes, in fact it's a good idea to have a dark mode for daytime, and a dark mode for night/dark ambient conditions. Also, blue light is a principle components of glare.

I'd like to show you some test samples and get your feedback if you're interested.

**Reply by thiannunawi** [reply]
_from_

**Comment by Bless360o** [comment]
> Yes, in fact it's a good idea to have a dark mode for daytime, and a dark mode for night/dark ambient conditions. Also, blue light is a principle components of glare.
> 
> I'd like to show you some test samples and get your feedback if you're interested.

Good day, thank you for reaching out to me.

Yes, absolutely. I'd be interested in helping out.

Is it possible to send samples through GitHub? Or better yet, here are my addresses specific to this situation -

autisticfriendlyreview@gmail.com

https://t.me/AUTISTICfriendly

Looking forward to co-creating abundance with you,


Cristina Esparragó 

Stay blessed 🌟

**Comment by cehfisher** [comment]
Hi @GaiaLoveWarrior - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/56523#discussioncomment-6743912). Could you please confirm and close out this issue? Thank you! (cc @queenofcorgis)
